libs/qCC_db/src/ccPointCloud.cpp

```diff
@@ -2012,10 +2014,11 @@ bool ccPointCloud::applyGaussianFilterToRGB(PointCoordinateType sigma,
 		progressCb->update(0);
 	}
 
-	void* additionalParameters[4] {	reinterpret_cast<void*>(&sigma),
+	void* additionalParameters[5] {	reinterpret_cast<void*>(&sigma),
```

Comment:
```suggestion
	void* additionalParameters[] {	reinterpret_cast<void*>(&sigma),
```

I believe this should work